x/bank/types/codec.go

```diff
@@ -35,7 +36,7 @@ var (
 	//
 	// The actual codec used for serialization should be provided to x/staking and
 	// defined at the application level.
-	ModuleCdc = codec.NewHybridCodec(amino)
+	ModuleCdc = codec.NewHybridCodec(amino, types.NewInterfaceRegistry())
```

Comment:
Why is this kept while others were completely removed?

Reply:
It is still used. I removed only the ModuleCdc's that had zero usage